### DISCUSSION TITLE
Allow for hiding the Skin Editor UI

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#25382] [Ideas]
This is part 2 of the editor improvements I thought of.

Hide the UI: Allow people to hide the UI while also being able to adjust the placement of elements while it's hidden

- Makes it easier to use elements for various things
- Makes precise placements of elements easier
- Makes it easier to retrieve lost elements

I think this would make skinning and skin editting far more enjoyable, as people won't have to leave and re-enter the editor over and over again to get placements right. It would also make it possible to move something if you accidentally lose it behind the UI.

**Comment by peppy** [comment]
Please describe he improvement you're asking for specifically in the title. We don't need "part 1" and "part 2".

**Reply by WalmartWhiteCat** [reply]
Got it.
